docs/content/latest/yugabyte-cloud/cloud-security/cloud-users.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Database authorization in Yugabyte Cloud clusters
+linkTitle: Database authorization
+description: The default YugabyteDB users and roles available in Yugabyte Cloud clusters.
+headcontent:
+image: /images/section_icons/deploy/enterprise.png
+menu:
+  latest:
+    identifier: cloud-users
+    parent: cloud-security
+    weight: 30
+isTocNested: true
+showAsideToc: true
+---
+
+To manage database access and authorization, YugabyteDB uses [role-based access control](../../../secure/authorization) (RBAC), consisting of a collection of privileges on resources given to roles.
+
+Clusters in Yugabyte Cloud include a set of default users and roles in YSQL and YCQL.
+
+## YSQL default roles and users
+
+To view the roles in your cluster, enter the following command:
+
+```sql
+yugabyte=> \du
+```
+
+```output
+                                                         List of roles
+  Role name   |                         Attributes                         |                     Member of                      
+--------------+------------------------------------------------------------+----------------------------------------------------
+ admin        | Create role, Create DB, Bypass RLS                         | {yb_superuser}
+ postgres     | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
+ yb_extension | Cannot login                                               | {}
+ yb_superuser | Create role, Create DB, Cannot login, Bypass RLS           | {pg_read_all_stats,pg_signal_backend,yb_extension}
+ yugabyte     | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
+```
+
+The following table describes the default YSQL roles and users in Yugabyte Cloud clusters.
```

Comment:
This is a duplicate of what's in Aishwarya's PR for the main docs. We probably want to de-dupe these and have a table in only one place... The fact that yb_superuser is cloud-only is a wrench in the works, though. Is admin cloud-only as well?